src/control/SwerveController.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+#include "../kinematics/DiffDriveKinematics.h"
+#include "../kinematics/SwerveDriveKinematics.h"
+
+#include <map>
+
+namespace control {
+enum class DriveMode {
+	Normal,
+	TurnInPlace,
+	Crab,
+};
+
+static const std::map<DriveMode, std::string> driveModeStrings = {
+	{DriveMode::Normal, "Normal"},
+	{DriveMode::TurnInPlace, "TurnInPlace"},
+	{DriveMode::Crab, "Crab"}};
```

Comment:
Instead of a global map, I think it would be a bit cleaner to provide an overload of the [`to_string()`](https://github.com/huskyroboticsteam/Resurgence/blob/master/src/utils/core.h#L50-L53) function.

To do that, do something like
```c++
namespace util {
std::string to_string(control::DriveMode mode);
}
```

See an example [here](https://github.com/huskyroboticsteam/Resurgence/blob/master/src/world_interface/data.h#L305-L307)

PS: `static` globals and methods in header-files are kind of a code smell. It will just be compiled separately everywhere the header is included, and all it does is prevent a [ODR](https://en.cppreference.com/w/cpp/language/definition) violation. If we wanted to put this map here, we should declare it as `extern const` and provide the definition in `SwerveController.cpp`